tutorials/fundamentals/installation.md

```diff
@@ -233,30 +275,24 @@ menu](../assets/tutorials/installation/windows_launch_napari.png)
 The very first time you launch napari the startup time will be fairly slow, but
 after that you will find that napari launches more quickly.
 
-
-### installing the Linux bundle
+### Installing the Linux bundle
 
 (Guide coming soon... In the meantime, if you try it and encounter issues, see
 below for how to contact us.)
+<!-- #endregion -->
 
-## bug reports
-
-If you are running into issues, please open a new issue on our [issue
-tracker](https://github.com/napari/napari/issues) and include the output of the
```

Comment:
Did we lose this content on getting help if needed, or has it just been moved elsewhere in the document? I think the link for where to submit an issue is useful information.

Reply:
this has been moved to the bottom under "Next steps".